containers/session-containers/skaha-desktop/src/common/scripts/vnc_startup.sh

```diff
@@ -37,9 +37,6 @@ Fore more information see: https://github.com/ConSol/docker-headless-vnc-contain
 if [[ $1 =~ -h|--help ]]; then
     help
     exit 0
-"vnc_startup.sh" 122L, 4274C
-    help
-    exit 0
```

Comment:
It would be better if no change to the container was made.  Is this necessary?  What does it accomplish?

Reply:
This was the section of code that I mentioned to you in slack that I did not understand what it was doing and thought it was extra code. Just want to clean up the code. Not related to my other changes.